riva/proto/riva_nmt.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+//
+// NVIDIA CORPORATION and its licensors retain all intellectual property
+// and proprietary rights in and to this software, related documentation
+// and any modifications thereto.  Any use, reproduction, disclosure or
+// distribution of this software and related documentation without an express
+// license agreement from NVIDIA CORPORATION is strictly prohibited.
+
+syntax="proto3";
+
+package nvidia.riva.nmt;
+
+option cc_enable_arenas = true;
+option go_package = "nvidia.com/riva_speech";
+
+/*
+ *  RivaTranslation service provides rpcs to translate between languages.
+ *
+ */
+service RivaTranslation {
+
+  // Translate text to text, from a source to a target language.  Currently source language is required, along with the model name.
+  // Multiple texts may be passed per request up to the given batch size for the model, which is set at translation pipeline creation time.
+rpc TranslateText(TranslateTextRequest) returns (TranslateTextResponse) {}
```

Comment:
Why do we have target lang in TranslateTextRequest if it is not required as per the comments above ?

Reply:
both are required to work with multi lingual models today.   source was called out here since wed like to make it optional in the future via language detection.

Reply:
I think the comments should not be based on what might be available in future. It just adds to the confusion. I think we should be consistent for all the fields in the comments which are mandatory in request and response based on what is available atm. We can update the comments and add optional field in future when we revisit this.